frb_codegen/Cargo.toml

```diff
@@ -51,3 +51,7 @@ bin-dir = "{bin}{binary-ext}"
 [package.metadata.binstall.overrides]
 x86_64-pc-windows-msvc = { pkg-fmt = "zip" }
 i686-pc-windows-msvc = { pkg-fmt = "zip" }
+
+[features]
+default = []
+chrono = []
```

Comment:
I guess we can always enable the chrono feature for frb_codegen, since the codegen is a binary, not a library (like frb_rust), and adding chrono will not add a lot to that binary size.

Reply:
modified in [5c001fc](https://github.com/fzyzcjy/flutter_rust_bridge/pull/694/commits/5c001fcbc2d09c11bddf2a90092ec6680080651b) and [f5fdc4a](https://github.com/fzyzcjy/flutter_rust_bridge/pull/694/commits/f5fdc4a7e9676588e19aefdb93320f6368fb3eca)